### PR TITLE
Updated the configuration/langen for consistency.

### PR DESCRIPTION
**Pull request by DerekRushton**

### CODE REVIEW

File: connectors/tanium/configurations/config.json [hunk 1]
```diff
@@ -1,27 +1,34 @@
 {
-    "connection": {
```
Comment (mdazam1942) [inline connectors/tanium/configurations/config.json]: this one looks like just formatting update.can you revert back these changes?

File: connectors/aws/configurations/lang_en.json [hunk 2]
```diff
@@ -29,12 +29,12 @@
                 "description": "Account ID of Amazon Web Service for authentication"
             },
             "aws_access_key_id": {
-                "label": "Client ID",
-                "description": "Client ID of Amazon Web Service for authentication"
+                "label": "AWS access key ID",
+                "description": "AWS Access Key ID is required for both AWS key-based and role-based authentication"
             },
             "aws_secret_access_key": {
-                "label": "Client secret",
```
Comment (mdazam1942) [inline connectors/aws/configurations/lang_en.json:36]: Not sure why are these fields called Client ID and secret! can you please verify the reason?

Comment (DerekRushton) [reply connectors/aws/configurations/lang_en.json:36]: I'm not sure why they originally had those names. However it is using boto3.

CAR code
```
    def getClient(self, service_name):
        return boto3.client(service_name, aws_access_key_id=self.aws_access_key_id,
                                   aws_secret_access_key=self.aws_secret_access_key,
                                   region_name=self.region_name)
```

```
    def getResource(self, service_name):
        return boto3.resource(service_name, aws_access_key_id=self.aws_access_key_id,
                                   aws_secret_access_key=self.aws_secret_access_key,
                                   region_name=self.region_name)
```

Which has documentation here https://boto3.amazonaws.com/v1/documentation/api/latest/guide/credentials.html 

that specifies that it needs AWS access/secret keys.

File: connectors/tanium/configurations/config.json [hunk 3]
```diff
@@ -7,14 +7,14 @@
         },
         "host": {
             "type": "text",
-            "regex": "^(([a-zA-Z0-9]|[a-zA-Z0-9][a-zA-Z0-9\\-]*[a-zA-Z0-9])\\.)*([A-Za-z0-9]|[A-Za-z0-9][A-Za-z0-9\\-]*[A-Za-z0-9])$",
-            "optional": true,
```
Comment (DerekRushton) [inline connectors/tanium/configurations/config.json:11]: I cleared this up with @omkar-g. This field shouldn't have been optional.

File: connectors/tanium/configurations/lang_en.json [hunk 4]
```diff
@@ -1,8 +1,8 @@
 {
     "connection": {
         "host": {
-            "label": "Management IP address or Hostname",
-            "description": "Specify the IP address or hostname of the data source so that IBM Cloud Pak for Security can communicate with it"
+            "label": "Management IP address or hostname",
+            "description": "Specify the IP address or  hostname of the data source so that IBM Cloud Pak for Security can communicate with it"
```
Comment (mdazam1942) [inline connectors/tanium/configurations/lang_en.json]: i think you should also remove the mention of "IBM Cloud Pak for Security" from  descriptions

File: connectors/aws/configurations/lang_en.json (hunk 2 above)
Comment (omkar-g) [reply connectors/aws/configurations/lang_en.json:36]: @DerekRushton I don't think `aws_access_key_id ` and `aws_secret_access_key ` are optional.

**Review by mdazam1942 — APPROVED** [review]

**Pull request merged**